apps/web/src/services/dashboards/components/DashboardCreateStep1SearchFilter.vue

```diff
@@ -152,13 +152,13 @@ const handleChangePluginFilter = (selected: PluginReferenceItem[]) => {
             </div>
             <div class="label">
                 <p-field-title class="title">
-                    Serivce
+                    Label
```

Comment:
question: No need to put translation here?

Reply:
Oh.. I'll do..